kdc/kdc.h

```diff
@@ -67,6 +67,8 @@ typedef struct krb5_kdc_configuration {
     krb5_boolean svc_use_strongest_session_key;
     krb5_boolean use_strongest_server_key;
 
+    krb5_boolean always_include_salt;
```

Comment:
I would move this to directly under `encode_as_rep_as_tgs_rep` and add a similar bug compatibility comment

Reply:
Thanks, will do